plugins/catalog/src/data/utils.tsx

```diff
@@ -23,7 +34,18 @@ export function envelopeToComponent(
   return {
     name: envelope.metadata?.name ?? '',
```

Comment:
both metadata and name are always present (so you can also remove the question mark on all metadata below)